Resources/Prototypes/Entities/Structures/Doors/blinds.yml

```diff
@@ -0,0 +1,142 @@
+- type: entity
+  id: BaseStructureBlinds
+  name: blinds
+  abstract: true
+  placement:
+    mode: SnapgridCenter
+    snap:
+    - Wallmount
+  components:
+  - type: WallMount
+  - type: Transform
+    anchored: true
+  - type: Clickable
+  - type: Sprite
+  - type: Physics
+    canCollide: true
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 100
+      behaviors:
+      - !type:DoActsBehavior
+        acts: ["Destruction"]
+  - type: IconSmooth
+    key: walls
+    mode: NoSprite
+  - type: BlockWeather
+  - type: Occluder
+
+- type: entity
+  id: BaseBlinds
+  name: blinds
+  parent: BaseStructureBlinds
+  abstract: true
+  description: You will definitely want to hide your hidden secrets behind them.
+  placement:
+    mode: SnapgridCenter
+    snap:
+    - Wallmount
+  components:
+  - type: Appearance
+  - type: Sprite
+    sprite: Structures/Doors/Blinds/wood_blinds.rsi
+    drawdepth: BlastDoors
+    layers:
+    - state: closed
+      map: ["enum.DoorVisualLayers.Base"]
+  - type: AnimationPlayer
+  - type: NavMapDoor
+  - type: Door
+    openDrawDepth: BlastDoors
+    closedDrawDepth: BlastDoors
+    bumpOpen: false
+    clickOpen: true
+    closeTimeOne: 0
+    closeTimeTwo: 0
+    openTimeOne: 0
+    openTimeTwo: 0
+    crushDamage:
+      types:
+        Blunt: 5
+    openSound:
+      path: /Audio/Effects/curtain_openclose.ogg
+    closeSound:
+      path: /Audio/Effects/curtain_openclose.ogg
+
+- type: entity
+  id: BlindsWoodFrame
+  name: wooden blinds frame
+  parent: BaseStructureBlinds
+  components:
+  - type: Sprite
+    sprite: Structures/Doors/Blinds/wood_blinds.rsi
+    state: frame
+  - type: Damageable
+    damageContainer: StructuralInorganic
+    damageModifierSet: Wood
+  - type: Construction
+    graph: Blinds
+    node: frameWood
```

Comment:
I believe you'll need to set deconstructionTarget on each of these to the appropriate base construction node.